### ISSUE TITLE
traffic.out not found

### ISSUE THREAD

**Issue by HugManh** [apache/trafficserver#11806]
Hi everyone,
I using TrafficSrever branch: master - [#828b7d3](https://github.com/apache/trafficserver/commit/828b7d3e40655982f8abc88a5e634ed17ec39681)
I added config to record.config, config as bellow:
```
CONFIG proxy.config.dump_mem_info_frequency INT 1
CONFIG proxy.config.res_track_memory INT 1
```
But when start trafficserver then traffic.out not found in ats/var/log/trafficserver

**To Reproduce**
```bash
autoreconf -i --force
./configure --prefix=/home/user/trafficserver/ats --enable-experimental-plugin
make && make install
/etc/init.d/trafficserver
```

**Comment by brbzull0** [comment]
Hi @HugManh.

ATS 10 is no longer using `records.config`, instead it uses [records.yaml](https://docs.trafficserver.apache.org/en/latest/admin-guide/files/records.yaml.en.html#records-yaml) now.  If you need to convert your old `records.config` to yaml, please follow [this](https://docs.trafficserver.apache.org/en/latest/admin-guide/tools/converting-records-to-yaml.en.html#converting-records-config-to-records-yaml).

Also, we've moved to cmake build system, please check [this](https://docs.trafficserver.apache.org/en/latest/admin-guide/installation/index.en.html#admin-installing) out.

Regards.

**Comment by bryancall** [comment]
I would agree with @brbzull0 that you should move to the 10.0.0 release instead of using checkout of master.

**Comment by HugManh** [comment]
Thank, @bryancall @brbzull0 
Sorry for the late reply, I will try switching to the 10.0.0 release.
